Misc/Positions_Area Positions 1 UK Permanent.txt

```diff
@@ -58,7 +58,7 @@ LTC_M_CTR:London Control:199.998:LMEN:L:LTC:CTR:-:-::
 MAN_M_CTR:Scottish Control:199.998:LMEN:L:MAN:CTR:-:-::
 
 SCO_M_CTR:Scottish Control:199.998:SM:L:SCO:CTR:-:-::
-STC_M_CTR:London Control:199.998:SM:L:SCO:CTR:-:-::
+STC_M_CTR:London Control:199.998:SM:L:STC:CTR:-:-::
```

Comment:
```suggestion
STC_M_CTR:Scottish Control:199.998:SM:L:STC:CTR:-:-::
```
Surely the name should be Scottish not London (not sure if this is used for anything though).